packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -97,17 +119,22 @@ describe("Garbage Collection Tests", () => {
             getLastSummaryTimestampMs: () => Date.now(),
         });
     }
+    let gc: GcWithPrivates | undefined;
```

Comment:
After reading through the PR, I don't understand why you made this change

Reply:
For the `gc?.dispose()` in `afterEach`